+ source ../common/common_lib.sh
++ source /root/GitRepo/mugen-riscv/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/GitRepo/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ grep expect
+ rpm -qa
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Thu Dec 29 17:57:15 2022 - INFO  - Start environmental preparation.
+ DNF_INSTALL lvm2-lockd
+ pkgs=lvm2-lockd
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/GitRepo/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs lvm2-lockd --node 1 --tempfile ''
Thu Dec 29 17:57:25 2022 - ERROR - pkgs:(lvm2-lockd) not found
+ tmpfile2=
+ '[' -z '' ']'
+ tmpfile=
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Thu Dec 29 17:57:26 2022 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Thu Dec 29 17:57:27 2022 - INFO  - Start testing...
+ test_execution lvmlockd.service
+ service=lvmlockd.service
++ date '+%Y-%m-%d %T'
+ log_time='2022-12-29 17:57:27'
+ test_restart lvmlockd.service
+ service=lvmlockd.service
+ systemctl restart lvmlockd.service
Failed to restart lvmlockd.service: Unit lvmlockd.service not found.
+ CHECK_RESULT 5 0 0 'lvmlockd.service restart failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='lvmlockd.service restart failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'lvmlockd.service restart failed'
+ LOG_ERROR 'lvmlockd.service restart failed'
+ message='lvmlockd.service restart failed'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'lvmlockd.service restart failed'
Thu Dec 29 17:57:28 2022 - ERROR - lvmlockd.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 34'
+ message='../common/common_lib.sh line 34'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 34'
Thu Dec 29 17:57:29 2022 - ERROR - ../common/common_lib.sh line 34
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status lvmlockd.service
+ grep 'Active: active'
Unit lvmlockd.service could not be found.
+ CHECK_RESULT 1 0 0 'lvmlockd.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='lvmlockd.service restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'lvmlockd.service restart failed'
+ LOG_ERROR 'lvmlockd.service restart failed'
+ message='lvmlockd.service restart failed'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'lvmlockd.service restart failed'
Thu Dec 29 17:57:35 2022 - ERROR - lvmlockd.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 37'
+ message='../common/common_lib.sh line 37'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 37'
Thu Dec 29 17:57:36 2022 - ERROR - ../common/common_lib.sh line 37
+ return 0
+ systemctl stop lvmlockd.service
Failed to stop lvmlockd.service: Unit lvmlockd.service not loaded.
+ CHECK_RESULT 5 0 0 'lvmlockd.service stop failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='lvmlockd.service stop failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'lvmlockd.service stop failed'
+ LOG_ERROR 'lvmlockd.service stop failed'
+ message='lvmlockd.service stop failed'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'lvmlockd.service stop failed'
Thu Dec 29 17:57:37 2022 - ERROR - lvmlockd.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 39'
+ message='../common/common_lib.sh line 39'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 39'
Thu Dec 29 17:57:38 2022 - ERROR - ../common/common_lib.sh line 39
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status lvmlockd.service
+ grep 'Active: inactive'
Unit lvmlockd.service could not be found.
+ CHECK_RESULT 1 0 0 'lvmlockd.service stop failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='lvmlockd.service stop failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'lvmlockd.service stop failed'
+ LOG_ERROR 'lvmlockd.service stop failed'
+ message='lvmlockd.service stop failed'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'lvmlockd.service stop failed'
Thu Dec 29 17:57:45 2022 - ERROR - lvmlockd.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 42'
+ message='../common/common_lib.sh line 42'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 42'
Thu Dec 29 17:57:46 2022 - ERROR - ../common/common_lib.sh line 42
+ return 0
+ systemctl start lvmlockd.service
Failed to start lvmlockd.service: Unit lvmlockd.service not found.
+ CHECK_RESULT 5 0 0 'lvmlockd.service start failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='lvmlockd.service start failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'lvmlockd.service start failed'
+ LOG_ERROR 'lvmlockd.service start failed'
+ message='lvmlockd.service start failed'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'lvmlockd.service start failed'
Thu Dec 29 17:57:46 2022 - ERROR - lvmlockd.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 44'
+ message='../common/common_lib.sh line 44'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 44'
Thu Dec 29 17:57:47 2022 - ERROR - ../common/common_lib.sh line 44
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status lvmlockd.service
+ grep 'Active: active'
Unit lvmlockd.service could not be found.
+ CHECK_RESULT 1 0 0 'lvmlockd.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='lvmlockd.service start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'lvmlockd.service start failed'
+ LOG_ERROR 'lvmlockd.service start failed'
+ message='lvmlockd.service start failed'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'lvmlockd.service start failed'
Thu Dec 29 17:57:54 2022 - ERROR - lvmlockd.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 47'
+ message='../common/common_lib.sh line 47'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 47'
Thu Dec 29 17:57:55 2022 - ERROR - ../common/common_lib.sh line 47
+ return 0
+ test_enabled lvmlockd.service
+ service=lvmlockd.service
++ systemctl is-enabled lvmlockd.service
Failed to get unit file state for lvmlockd.service: No such file or directory
+ state=
+ '[' '' == enabled ']'
+ '[' '' == disabled ']'
+ '[' '' == masked ']'
+ '[' '' == static ']'
+ LOG_INFO 'Unit is indirect, ignoring.'
+ message='Unit is indirect, ignoring.'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Unit is indirect, ignoring.'
Thu Dec 29 17:57:56 2022 - INFO  - Unit is indirect, ignoring.
+ journalctl --since '2022-12-29 17:57:27' -u lvmlockd.service
+ grep -i 'fail\|error'
+ grep -v -i 'DEBUG\|INFO\|WARNING'
+ CHECK_RESULT 1 0 1 'There is an error message for the log of lvmlockd.service'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of lvmlockd.service'
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ test_reload lvmlockd.service
+ service=lvmlockd.service
+ systemctl start lvmlockd.service
Failed to start lvmlockd.service: Unit lvmlockd.service not found.
+ systemctl reload lvmlockd.service
+ grep 'Job type reload is not applicable'
+ CHECK_RESULT 1 0 0 'Job type reload is not applicable for unit lvmlockd.service'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Job type reload is not applicable for unit lvmlockd.service'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Job type reload is not applicable for unit lvmlockd.service'
+ LOG_ERROR 'Job type reload is not applicable for unit lvmlockd.service'
+ message='Job type reload is not applicable for unit lvmlockd.service'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Job type reload is not applicable for unit lvmlockd.service'
Thu Dec 29 17:57:57 2022 - ERROR - Job type reload is not applicable for unit lvmlockd.service
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 91'
+ message='../common/common_lib.sh line 91'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 91'
Thu Dec 29 17:57:57 2022 - ERROR - ../common/common_lib.sh line 91
+ return 0
+ systemctl status lvmlockd.service
+ grep 'Active: active'
Unit lvmlockd.service could not be found.
+ systemctl status lvmlockd.service
+ grep 'inactive (dead)'
Unit lvmlockd.service could not be found.
+ return 1
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Thu Dec 29 17:57:58 2022 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 7
+ test 7 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Thu Dec 29 17:57:59 2022 - ERROR - failed to execute the case.
+ exit 7
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Thu Dec 29 17:58:00 2022 - INFO  - start environment cleanup.
+ systemctl stop lvmlockd.service
Failed to stop lvmlockd.service: Unit lvmlockd.service not loaded.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z '' ]]
+ [[ -z '' ]]
+ LOG_WARN 'no thing to do.'
+ message='no thing to do.'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level warn --message 'no thing to do.'
Thu Dec 29 17:58:01 2022 - WARN  - no thing to do.
+ return 0
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Thu Dec 29 17:58:01 2022 - INFO  - Finish environment cleanup!
